docs/UserGuide.md

```diff
@@ -213,20 +219,24 @@ There cannot be an already existing tag with the same name; tags must be unique.
 
 </div>
 
-### Deleting a task: `deletet`
+### Listing tasks : `listt`
 
-Deletes the specified task from the task list.
+Shows a list of all the tasks in the task list as per the specified filtering options. `listt` has the three folowing formats:
 
-Format: `deletet 1`
+Format: `listt all/`
 
-* Deletes the task at the specified `INDEX`.
-* The index refers to the index number shown in the displayed task list.
-* The index must be a positive integer 1, 2, 3, …​
+Shows a list of tasks that is marked as completed in the task list.
 
-Examples:
-* `deletet 1` Deletes the task at index 1.
-* `deletet 2` Deletes the task at index 2.
+Format: `listt c/`
 
+Shows a list of tasks that is not mark as completed in the task list.
+
+Format: `listt nc/`
+
+<div markdown="span" class="alert alert-warning">:bulb: **Tip**
+Good usage of `listt` will help in keeping track of tasks by status!
+
+</div>
```

Comment:
Feels weird @junjunjieOng 

Do you think its better if its:

Format:

1. a
2. b
3. c

Reply:
If is easier for user to read, sure can do with this format👌